### DISCUSSION TITLE
Change reconnection after initial connection

### DISCUSSION THREAD

**Discussion by markg85** [socketio/socket.io#3871] [Q&A] [answered]
Hi,

I have a list of URL's i want to try and make a connection too.
I'm iterating a list of url's (via promises) and use the first promise that resolves as the connection i want to have.
For that, i set the reconnection flag initially to false so that socket.io only tries a connection once.

Once i have a working connection to one of the URL's i want to flip the reconnection flag to true.
How do i do that?
I'm fine if it's a super hacky solution.

Cheers,
Mark

**Comment by darrachequesne** [comment, marked as answer]
You can simply update the `reconnection` attribute of the Manager:

```js
socket.io.reconnection(true);
```

Documentation: https://socket.io/docs/v4/client-api/#manager-reconnection-value

**Reply by markg85** [reply]
Hah, and i couldn't find it.. :)
That easy. And works exactly as intended. Awesome, thank you for the tip!
